src/app/content/components/TableOfContents/index.tsx

```diff
@@ -150,51 +156,106 @@ function maybeAriaLabel(page: LinkedArchiveTreeSection) {
     return {};
   }
 
-  return {'aria-label': `${titleText} - Chapter ${parentNum}`};
+  return { 'aria-label': `${titleText} - Chapter ${parentNum}` };
+}
+
+function ArchiveTreeComponent({
+  item,
+  book,
+  page,
+  activeSection,
+  onNavigate,
+}: {
+  item: LinkedArchiveTree;
+  book: Book | undefined;
+  page: Page | undefined;
+  activeSection: React.RefObject<HTMLElement>;
+  onNavigate: () => void;
+}) {
+  const sectionType = getArchiveTreeSectionType(item);
+  const [isOpen, setOpen] = React.useState<boolean>(shouldBeOpen(page, item));
+  const toggleOpen = (event: React.MouseEvent<HTMLDivElement, MouseEvent> | React.KeyboardEvent<HTMLDivElement>) => {
+    if ('key' in event) {
+      if (event.key !== 'Enter' && event.key !== ' ') {
```

Comment:
i believe this needs a `preventDefault` and i think you may be required to add support for the additional keyboard operations defined https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/#kbd_label

you may actually be closer to achieving the disclosure pattern https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/ which i think would also be an acceptable solution here, it just requires different roles